nodecg-io-core/dashboard/crypto.ts

```diff
@@ -83,15 +85,15 @@ export async function sendAuthenticatedMessage<V>(messageName: string, message:
 }
 
 /**
- * Returns whether a password has been set in the crypto module aka. whether is is authenticated.
+ * Returns whether a password has been set in the crypto module aka. whether is authenticated.
```

Comment:
The first `is` was most likely an typo of `it`.
```suggestion
 * Returns whether a password has been set in the crypto module aka. whether it is authenticated.
```